nemo/utils/config_utils.py

```diff
@@ -17,10 +17,10 @@
 
 from omegaconf import DictConfig, OmegaConf, open_dict
 
-from nemo.core.config.modelPT import ModelPTConfig
+from nemo.core.config.modelPT import ModelConfig
 
 
-def update_model_config(model_cls: ModelPTConfig, update_cfg: DictConfig, drop_missing_subconfigs: bool = True):
+def update_model_config(model_cls: ModelConfig, update_cfg: DictConfig, drop_missing_subconfigs: bool = True):
```

Comment:
This is incorrect. The update method is built with the `NeMoModelConfig` template in mind, and will break if its passed the inner model. Please revert.

Reply:
Or if not, please update the code and helpers here to use the inner model rather inside the ModelConfig instead of assuming its a ModelPTConfig/NeMoModelConfig.

Reply:
good catch, changed to NeMoConfig